AlfrescoApp/Model/Managers/RealmSyncManager.m

```diff
@@ -760,6 +767,70 @@ - (void)didUploadNewVersionForDocument:(AlfrescoDocument *)document updatedDocum
     }
 }
 
+- (void)addNodeToSync:(AlfrescoNode *)node withCompletionBlock:(void (^)(BOOL completed))completionBlock
+{
+    UserAccount *selectedAccount = [[AccountManager sharedManager] selectedAccount];
+    NSString *selectedAccountIdentifier = selectedAccount.accountIdentifier;
+    NSMutableDictionary *nodesInfoForSelectedAccount = self.syncNodesInfo[selectedAccountIdentifier];
+    NSMutableArray *topLevelSyncNodes = nodesInfoForSelectedAccount[selectedAccountIdentifier];
+    BOOL isSyncNodesInfoInMemory = (topLevelSyncNodes != nil);
```

Comment:
Would be better to read the top level sync nodes from RealmDB as syncNodesInfo dictionary is not used anywhere else so far in the manager (thus will hold no data)